src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Altinn.Platform.Authorization.csproj

```diff
@@ -19,6 +21,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="WindowsAzure.Storage" Version="9.3.3" />
```

Comment:
I don't see that we use this reference in storage. Why are we referencing it here?

Reply:
It is used for the cloudBlobClient in the PolicyRepository.

Reply:
ok, but do we need an explicit reference for that in project file? Storage also uses the CloudBlobClient but doesn't seem to have a reference in project file. So I wonder if we could remove this.

Reply:
storage has a reference in the project file to     <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="9.4.2" /> which we won't need since we have this reference.. tried testing without, but there's no obvious way around it.

Reply:
Which reference is that in storage?